internal/api/handlers.go

```diff
@@ -34,6 +34,7 @@ func NewGoCertRouter(env *Environment) http.Handler {
 	apiV1Router.HandleFunc("GET /accounts/{id}", GetUserAccount(env))
 	apiV1Router.HandleFunc("GET /accounts", GetUserAccounts(env))
 	apiV1Router.HandleFunc("POST /accounts", PostUserAccount(env))
+	apiV1Router.HandleFunc("DELETE /accounts/{id}/account", DeleteUserAccount(env))
```

Comment:
```suggestion
	apiV1Router.HandleFunc("DELETE /accounts/{id}", DeleteUserAccount(env))
```

Reply:
I see rebasing didn't go perfectly :D Thanks for catching it.